tests/testthat/test_utils.r

```diff
@@ -13,10 +11,12 @@ test_that('skip_repeated does not skip three or less consecutive items', {
 })
 
 test_that('skip_repeated works on tracebacks', {
-    err <- try_capture_stack(quote({
+    err <- evaluate::try_capture_stack(quote({
         f <- function(x) stop(x)
         f(1)
     }), new.env())
     skipped_stack <- skip_repeated(err$calls)
-    expect_is(skipped_stack[[1]], 'call')
+    # See discussion in #741 about using inherits() here and not
+    #   expect_type(., "language") or expect_true(is.call(.)) or expect_mode(., "call")
+    expect_true(inherits(skipped_stack[[1]], 'call'))
```

Comment:
I think you could make this test more clear by making it more explicit:

```R
expect_equal(skipped_stack[[1]], quote(f(1)))
```

Or even:

```R
expect_equal(skipped_stack, list(quote(f(1)), quote(stop(x))))
```

That illustrates a subtle problem somewhere in the stack because if you print `skipped_stack` you see:

```
[[1]]
f(1)

[[2]]
function(x) stop(x)
```

Because the srcref is out of sync with the actual contents of the element.